build-support/bin/pre-commit.sh

```diff
@@ -56,8 +56,7 @@ echo "* Checking for bad shell patterns" && ./build-support/bin/check_shell.sh |
 # When travis builds a tag, it does so in a shallow clone without master fetched, which
 # fails in pants changed.
 if git rev-parse --verify "master" &>/dev/null; then
-  echo "* Checking imports" && ./build-support/bin/isort.sh || \
-    die "To fix import sort order, run \`\"$(pwd)/build-support/bin/isort.sh\" -f\`"
+  echo "* Checking imports" && ./build-support/bin/isort.py || exit 1
```

Comment:
So this is a fantastic change...how do you feel about adding a test in https://github.com/pantsbuild/pants/blob/master/tests/python/pants_test/repo_scripts/test_git_hooks.py? It's much easier to do this sort of thing as we refactor the repo scripts, I think. Since we're relying on pants to do changed detection instead of piping in a git diff, we can probably avoid some of the more annoying git repo setup this time.

Reply:
Will look into how to do this today!